make/test.mk

```diff
@@ -50,6 +50,16 @@ test-e2e: prepare-e2e verify-migration-and-deploy-e2e e2e-run-parallel e2e-run
 test-e2e-without-migration: prepare-e2e deploy-e2e e2e-run-parallel e2e-run
 	@echo "To clean the cluster run 'make clean-e2e-resources'"
 
+.PHONY: test-e2e-no-parallel
+## Run the e2e tests without migration tests
```

Comment:
Copy-paste from a different target. It seems that this is actually skipping both parallel and migration tests.

Reply:
fixed